content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+* Other open source engines like Godot have had a lot of success with MIT-only licensing
+
+However there were a variety of issues that have come up that make dual-licensing Bevy under both MIT and Apache-2.0 compelling:
+
+* The MIT license (arguably) requires binaries to reproduce countless copies of the same license boilerplate for every MIT library in use. Apache-2.0 allows us to compress the boilerplate into a single instance of the license.
+* The Apache-2.0 license has protections from patent trolls and an explicit contribution licensing clause.
+* The Rust ecosystem is largely Apache-2.0. Being available under that license is good for interoperation and opens the doors to upstreaming Bevy code into other projects (Rust, the async ecosystem, etc).
+* The Apache license is incompatible with GPLv2, but MIT is compatible. 
+
+## Bevy Org Changes
+
+### More pull request mergers!
+
+I've been at my scalability limits for awhile. It has been  * cough * ... challenging  ... to build the engine features I need to, review every single pull request quickly, and preserve my mental health. I've made it this far ... sometimes by overworking myself and sometimes by letting PRs sit unmerged for longer than I'd like. By scaling out, we can have our cake and eat it too!
+
+* @mockersf now has merge rights for "uncontroversial changes"
+* @alice-i-cecile now has merge rights for "uncontroversial documentation changes"
+
+### New issue labels
+
+After [much discussion about naming conventions and colors](https://github.com/bevyengine/bevy/issues/2256), we finally have a [fresh new set of issue labels](https://github.com/bevyengine/bevy/labels) (loosely inspired by the rust repo). The [Bevy Triage Team](https://github.com/orgs/bevyengine/teams/triage-team/members) can finally express themselves fully!
+
+### Comprehensive CONTRIBUTING.md
+
+<div class="release-feature-authors">authors: @alice-i-cecile</div>
+
+We now have a relatively complete [Contributors Guide](https://github.com/bevyengine/bevy/blob/main/CONTRIBUTING.md). If you are interested in contributing code or documentation to Bevy, that is a great place to start! 
+
+### CI Build System Improvements
+
+<div class="release-feature-authors">authors: @mockersf, @NathanSWard, @NiklasEi</div>
+
+We made a ton of CI improvements this release:
+
+* We now fail on cargo doc warnings
+* We now use [cargo deny](https://github.com/EmbarkStudios/cargo-deny) to protect against vulnerabilities, duplicate dependencies, and invalid licenses
+* PRs are now automatically labeled with the `S-Needs-Triage` label
+* Ci stability and speed improvements
+* We now check that our benchmarks build
+* We now assert compiler errors for compile_fail tests, giving us much stricter guarantees
+* Examples are now run using lavapipe (instead of swiftshader) for faster CI validation
+
+## What's Next For Bevy?
```

Comment:
Is it worth mentioning the RFCs here? Getting more eyes on some of the more further out features for the engine, particularly early on, would be super useful in scoping and shaping those features based on demand and feedback.